polish edit/update features

Garyguo2011 opened the pull request.

Garyguo2011 commented on app/models/user.rb:
```diff
@@ -8,6 +8,7 @@ class User < ActiveRecord::Base
   attr_accessible :email, :password, :password_confirmation, :remember_me,
   					:first_name, :last_name
   # attr_accessible :title, :body
+  validates :email, :uniqueness => true
```
Jinge please double check if email format is contain "@" and dot "."

spdemille-ucb commented:
hey gary, i managed to install skype on my laptop which has a camera and
microphone :) also, if you pull the calcentral-task-view you can see the
CalCentral CSS applied to the task we already have :) i'm working on the
task view and trying to figure out how to do graphs. i will push more soon
:)

-- Sterling


On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <notifications@github.com>
wrote:

> Merged #12
> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786>
> .
>

Garyguo2011 commented:
Looks nice. Just need some changes for create/update/edit page. :D

Thanks for you effort.

Gary

-- 
Xinran Guo
xguo@berkeley.edu

On March 9, 2015 at 7:06:14 PM, spdemille-ucb (notifications@github.com) wrote:

hey gary, i managed to install skype on my laptop which has a camera and
microphone :) also, if you pull the calcentral-task-view you can see the
CalCentral CSS applied to the task we already have :) i'm working on the
task view and trying to figure out how to do graphs. i will push more soon
:)

-- Sterling


On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <notifications@github.com>
wrote:

> Merged #12
> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786>
> .
>
—
Reply to this email directly or view it on GitHub.

spdemille-ucb commented:
what changes need to be done? the files look complete

On Mon, Mar 9, 2015 at 7:14 PM, Garyguo2011 <notifications@github.com>
wrote:

> Looks nice. Just need some changes for create/update/edit page. :D
>
> Thanks for you effort.
>
> Gary
>
> --
> Xinran Guo
> xguo@berkeley.edu
>
> On March 9, 2015 at 7:06:14 PM, spdemille-ucb (notifications@github.com)
> wrote:
>
> hey gary, i managed to install skype on my laptop which has a camera and
> microphone :) also, if you pull the calcentral-task-view you can see the
> CalCentral CSS applied to the task we already have :) i'm working on the
> task view and trying to figure out how to do graphs. i will push more soon
> :)
>
> -- Sterling
>
>
> On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <notifications@github.com>
> wrote:
>
> > Merged #12
> > <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12>.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786
> >
>
> > .
> >
> —
> Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77983958>
> .
>

Garyguo2011 commented:
Like we discuss last Friday, we probably need to keep edit/update in the same page of subtask view.
and probably a bit move compound around. like Remaining time etc.

Thanks
Gary

-- 
Xinran Guo
xguo@berkeley.edu

On March 9, 2015 at 7:47:21 PM, spdemille-ucb (notifications@github.com) wrote:

what changes need to be done? the files look complete

On Mon, Mar 9, 2015 at 7:14 PM, Garyguo2011 <notifications@github.com>
wrote:

> Looks nice. Just need some changes for create/update/edit page. :D
>
> Thanks for you effort.
>
> Gary
>
> --
> Xinran Guo
> xguo@berkeley.edu
>
> On March 9, 2015 at 7:06:14 PM, spdemille-ucb (notifications@github.com)
> wrote:
>
> hey gary, i managed to install skype on my laptop which has a camera and
> microphone :) also, if you pull the calcentral-task-view you can see the
> CalCentral CSS applied to the task we already have :) i'm working on the
> task view and trying to figure out how to do graphs. i will push more soon
> :)
>
> -- Sterling
>
>
> On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <notifications@github.com>
> wrote:
>
> > Merged #12
> > <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12>.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786
> >
>
> > .
> >
> —
> Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77983958>
> .
>
—
Reply to this email directly or view it on GitHub.

Garyguo2011 commented:
Hi Sterling,

Did you copy application.css as minimum file? (without any white space)


Thanks,
Gary

-- 
Xinran Guo
xguo@berkeley.edu

On March 9, 2015 at 8:11:56 PM, Xinran Guo (xguo@berkeley.edu) wrote:

Like we discuss last Friday, we probably need to keep edit/update in the same page of subtask view.
and probably a bit move compound around. like Remaining time etc.

Thanks
Gary

-- 
Xinran Guo
xguo@berkeley.edu

On March 9, 2015 at 7:47:21 PM, spdemille-ucb (notifications@github.com) wrote:

what changes need to be done? the files look complete

On Mon, Mar 9, 2015 at 7:14 PM, Garyguo2011 <notifications@github.com>
wrote:

> Looks nice. Just need some changes for create/update/edit page. :D
>
> Thanks for you effort.
>
> Gary
>
> --
> Xinran Guo
> xguo@berkeley.edu
>
> On March 9, 2015 at 7:06:14 PM, spdemille-ucb (notifications@github.com)
> wrote:
>
> hey gary, i managed to install skype on my laptop which has a camera and
> microphone :) also, if you pull the calcentral-task-view you can see the
> CalCentral CSS applied to the task we already have :) i'm working on the
> task view and trying to figure out how to do graphs. i will push more soon
> :)
>
> -- Sterling
>
>
> On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <notifications@github.com>
> wrote:
>
> > Merged #12
> > <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12>.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786
> >
>
> > .
> >
> —
> Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77983958>
> .
>
—
Reply to this email directly or view it on GitHub.

spdemille-ucb commented:
hi gary, yes, it was without whitespace. it was already that way from the
cal-central website :) i have added a couple more changes if you pull
calcentral-task-view again. i'm still trying to figure out the best way to
do the update/create/delete links for the subtasks. i think i have
something soon :) thx!

On Mon, Mar 9, 2015 at 10:04 PM, Garyguo2011 <notifications@github.com>
wrote:

> Hi Sterling,
>
> Did you copy application.css as minimum file? (without any white space)
>
>
> Thanks,
> Gary
>
> --
> Xinran Guo
> xguo@berkeley.edu
>
> On March 9, 2015 at 8:11:56 PM, Xinran Guo (xguo@berkeley.edu) wrote:
>
> Like we discuss last Friday, we probably need to keep edit/update in the
> same page of subtask view.
> and probably a bit move compound around. like Remaining time etc.
>
> Thanks
> Gary
>
> --
> Xinran Guo
> xguo@berkeley.edu
>
> On March 9, 2015 at 7:47:21 PM, spdemille-ucb (notifications@github.com)
> wrote:
>
> what changes need to be done? the files look complete
>
> On Mon, Mar 9, 2015 at 7:14 PM, Garyguo2011 <notifications@github.com>
> wrote:
>
> > Looks nice. Just need some changes for create/update/edit page. :D
> >
> > Thanks for you effort.
> >
> > Gary
> >
> > --
> > Xinran Guo
> > xguo@berkeley.edu
> >
> > On March 9, 2015 at 7:06:14 PM, spdemille-ucb (notifications@github.com)
> > wrote:
> >
> > hey gary, i managed to install skype on my laptop which has a camera and
> > microphone :) also, if you pull the calcentral-task-view you can see the
> > CalCentral CSS applied to the task we already have :) i'm working on the
> > task view and trying to figure out how to do graphs. i will push more
> soon
> > :)
> >
> > -- Sterling
> >
> >
> > On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <notifications@github.com>
> > wrote:
> >
> > > Merged #12
> > > <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12>.
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> >
> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786
> > >
> >
> > > .
> > >
> > —
> > Reply to this email directly or view it on GitHub.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77983958
> >
>
> > .
> >
> —
> Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77996500>
> .
>

spdemille-ucb commented:
Hi Gary, I've added create and update subtasks to the calcentral-task-view
branch :) still working on delete

On Tue, Mar 10, 2015 at 2:12 AM, Sterling P DE MILLE <spdemille@berkeley.edu
> wrote:

> hi gary, yes, it was without whitespace. it was already that way from the
> cal-central website :) i have added a couple more changes if you pull
> calcentral-task-view again. i'm still trying to figure out the best way to
> do the update/create/delete links for the subtasks. i think i have
> something soon :) thx!
>
> On Mon, Mar 9, 2015 at 10:04 PM, Garyguo2011 <notifications@github.com>
> wrote:
>
>> Hi Sterling,
>>
>> Did you copy application.css as minimum file? (without any white space)
>>
>>
>> Thanks,
>> Gary
>>
>> --
>> Xinran Guo
>> xguo@berkeley.edu
>>
>> On March 9, 2015 at 8:11:56 PM, Xinran Guo (xguo@berkeley.edu) wrote:
>>
>> Like we discuss last Friday, we probably need to keep edit/update in the
>> same page of subtask view.
>> and probably a bit move compound around. like Remaining time etc.
>>
>> Thanks
>> Gary
>>
>> --
>> Xinran Guo
>> xguo@berkeley.edu
>>
>> On March 9, 2015 at 7:47:21 PM, spdemille-ucb (notifications@github.com)
>> wrote:
>>
>> what changes need to be done? the files look complete
>>
>> On Mon, Mar 9, 2015 at 7:14 PM, Garyguo2011 <notifications@github.com>
>> wrote:
>>
>> > Looks nice. Just need some changes for create/update/edit page. :D
>> >
>> > Thanks for you effort.
>> >
>> > Gary
>> >
>> > --
>> > Xinran Guo
>> > xguo@berkeley.edu
>> >
>> > On March 9, 2015 at 7:06:14 PM, spdemille-ucb (notifications@github.com
>> )
>> > wrote:
>> >
>> > hey gary, i managed to install skype on my laptop which has a camera and
>> > microphone :) also, if you pull the calcentral-task-view you can see the
>> > CalCentral CSS applied to the task we already have :) i'm working on the
>> > task view and trying to figure out how to do graphs. i will push more
>> soon
>> > :)
>> >
>> > -- Sterling
>> >
>> >
>> > On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <notifications@github.com>
>> > wrote:
>> >
>> > > Merged #12
>> > > <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12>.
>> > >
>> > > —
>> > > Reply to this email directly or view it on GitHub
>> > > <
>> >
>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786
>> > >
>> >
>> > > .
>> > >
>> > —
>> > Reply to this email directly or view it on GitHub.
>> >
>> > —
>> > Reply to this email directly or view it on GitHub
>> > <
>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77983958
>> >
>>
>> > .
>> >
>> —
>> Reply to this email directly or view it on GitHub.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77996500>
>> .
>>
>
>

Garyguo2011 commented:
Nice, Thanks for the hard work.

Looks fantastic, Specially for the header bar. Super cool. Since you are helping (Daniel, and Roy)’s feature. Probably Your guys need to communicate with each other to avoid duplicate view work.

I figure out how to write about Rspec yesterday. will fill test cases correspond to every feature we come up with. i will check your branch. :D

Thanks,
Gary

-- 
Xinran Guo
xguo@berkeley.edu

On March 10, 2015 at 12:26:24 PM, spdemille-ucb (notifications@github.com) wrote:

Hi Gary, I've added create and update subtasks to the calcentral-task-view
branch :) still working on delete

On Tue, Mar 10, 2015 at 2:12 AM, Sterling P DE MILLE <spdemille@berkeley.edu
> wrote:

> hi gary, yes, it was without whitespace. it was already that way from the
> cal-central website :) i have added a couple more changes if you pull
> calcentral-task-view again. i'm still trying to figure out the best way to
> do the update/create/delete links for the subtasks. i think i have
> something soon :) thx!
>
> On Mon, Mar 9, 2015 at 10:04 PM, Garyguo2011 <notifications@github.com>
> wrote:
>
>> Hi Sterling,
>>
>> Did you copy application.css as minimum file? (without any white space)
>>
>>
>> Thanks,
>> Gary
>>
>> --
>> Xinran Guo
>> xguo@berkeley.edu
>>
>> On March 9, 2015 at 8:11:56 PM, Xinran Guo (xguo@berkeley.edu) wrote:
>>
>> Like we discuss last Friday, we probably need to keep edit/update in the
>> same page of subtask view.
>> and probably a bit move compound around. like Remaining time etc.
>>
>> Thanks
>> Gary
>>
>> --
>> Xinran Guo
>> xguo@berkeley.edu
>>
>> On March 9, 2015 at 7:47:21 PM, spdemille-ucb (notifications@github.com)
>> wrote:
>>
>> what changes need to be done? the files look complete
>>
>> On Mon, Mar 9, 2015 at 7:14 PM, Garyguo2011 <notifications@github.com>
>> wrote:
>>
>> > Looks nice. Just need some changes for create/update/edit page. :D
>> >
>> > Thanks for you effort.
>> >
>> > Gary
>> >
>> > --
>> > Xinran Guo
>> > xguo@berkeley.edu
>> >
>> > On March 9, 2015 at 7:06:14 PM, spdemille-ucb (notifications@github.com
>> )
>> > wrote:
>> >
>> > hey gary, i managed to install skype on my laptop which has a camera and
>> > microphone :) also, if you pull the calcentral-task-view you can see the
>> > CalCentral CSS applied to the task we already have :) i'm working on the
>> > task view and trying to figure out how to do graphs. i will push more
>> soon
>> > :)
>> >
>> > -- Sterling
>> >
>> >
>> > On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <notifications@github.com>
>> > wrote:
>> >
>> > > Merged #12
>> > > <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12>.
>> > >
>> > > —
>> > > Reply to this email directly or view it on GitHub
>> > > <
>> >
>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786
>> > >
>> >
>> > > .
>> > >
>> > —
>> > Reply to this email directly or view it on GitHub.
>> >
>> > —
>> > Reply to this email directly or view it on GitHub
>> > <
>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77983958
>> >
>>
>> > .
>> >
>> —
>> Reply to this email directly or view it on GitHub.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77996500>
>> .
>>
>
>
—
Reply to this email directly or view it on GitHub.

spdemille-ucb commented:
ok, yeah, i texted Roy about it and told him to copy what he needs :) i
just did a page view source and copy/paste lol :P

On Tue, Mar 10, 2015 at 12:37 PM, Garyguo2011 <notifications@github.com>
wrote:

> Nice, Thanks for the hard work.
>
> Looks fantastic, Specially for the header bar. Super cool. Since you are
> helping (Daniel, and Roy)’s feature. Probably Your guys need to communicate
> with each other to avoid duplicate view work.
>
> I figure out how to write about Rspec yesterday. will fill test cases
> correspond to every feature we come up with. i will check your branch. :D
>
> Thanks,
> Gary
>
> --
> Xinran Guo
> xguo@berkeley.edu
>
> On March 10, 2015 at 12:26:24 PM, spdemille-ucb (notifications@github.com)
> wrote:
>
> Hi Gary, I've added create and update subtasks to the calcentral-task-view
> branch :) still working on delete
>
> On Tue, Mar 10, 2015 at 2:12 AM, Sterling P DE MILLE <
> spdemille@berkeley.edu
> > wrote:
>
> > hi gary, yes, it was without whitespace. it was already that way from the
> > cal-central website :) i have added a couple more changes if you pull
> > calcentral-task-view again. i'm still trying to figure out the best way
> to
> > do the update/create/delete links for the subtasks. i think i have
> > something soon :) thx!
> >
> > On Mon, Mar 9, 2015 at 10:04 PM, Garyguo2011 <notifications@github.com>
> > wrote:
> >
> >> Hi Sterling,
> >>
> >> Did you copy application.css as minimum file? (without any white space)
> >>
> >>
> >> Thanks,
> >> Gary
> >>
> >> --
> >> Xinran Guo
> >> xguo@berkeley.edu
> >>
> >> On March 9, 2015 at 8:11:56 PM, Xinran Guo (xguo@berkeley.edu) wrote:
> >>
> >> Like we discuss last Friday, we probably need to keep edit/update in the
> >> same page of subtask view.
> >> and probably a bit move compound around. like Remaining time etc.
> >>
> >> Thanks
> >> Gary
> >>
> >> --
> >> Xinran Guo
> >> xguo@berkeley.edu
> >>
> >> On March 9, 2015 at 7:47:21 PM, spdemille-ucb (notifications@github.com
> )
> >> wrote:
> >>
> >> what changes need to be done? the files look complete
> >>
> >> On Mon, Mar 9, 2015 at 7:14 PM, Garyguo2011 <notifications@github.com>
> >> wrote:
> >>
> >> > Looks nice. Just need some changes for create/update/edit page. :D
> >> >
> >> > Thanks for you effort.
> >> >
> >> > Gary
> >> >
> >> > --
> >> > Xinran Guo
> >> > xguo@berkeley.edu
> >> >
> >> > On March 9, 2015 at 7:06:14 PM, spdemille-ucb (
> notifications@github.com
> >> )
> >> > wrote:
> >> >
> >> > hey gary, i managed to install skype on my laptop which has a camera
> and
> >> > microphone :) also, if you pull the calcentral-task-view you can see
> the
> >> > CalCentral CSS applied to the task we already have :) i'm working on
> the
> >> > task view and trying to figure out how to do graphs. i will push more
> >> soon
> >> > :)
> >> >
> >> > -- Sterling
> >> >
> >> >
> >> > On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <notifications@github.com
> >
> >> > wrote:
> >> >
> >> > > Merged #12
> >> > > <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12>.
> >> > >
> >> > > —
> >> > > Reply to this email directly or view it on GitHub
> >> > > <
> >> >
> >>
> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786
> >> > >
> >> >
> >> > > .
> >> > >
> >> > —
> >> > Reply to this email directly or view it on GitHub.
> >> >
> >> > —
> >> > Reply to this email directly or view it on GitHub
> >> > <
> >>
> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77983958
> >> >
> >>
> >> > .
> >> >
> >> —
> >> Reply to this email directly or view it on GitHub.
> >>
> >> —
> >> Reply to this email directly or view it on GitHub
> >> <
> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77996500
> >
>
> >> .
> >>
> >
> >
> —
> Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-78131241>
> .
>

spdemille-ucb commented:
i was trying to see if i could make the sub task and task view inside the
center column on the dashboard page of calcentral. i guess i can try
scanning the css document for the column width :)

On Tue, Mar 10, 2015 at 1:00 PM, Sterling P DE MILLE <spdemille@berkeley.edu
> wrote:

> ok, yeah, i texted Roy about it and told him to copy what he needs :) i
> just did a page view source and copy/paste lol :P
>
> On Tue, Mar 10, 2015 at 12:37 PM, Garyguo2011 <notifications@github.com>
> wrote:
>
>> Nice, Thanks for the hard work.
>>
>> Looks fantastic, Specially for the header bar. Super cool. Since you are
>> helping (Daniel, and Roy)’s feature. Probably Your guys need to communicate
>> with each other to avoid duplicate view work.
>>
>> I figure out how to write about Rspec yesterday. will fill test cases
>> correspond to every feature we come up with. i will check your branch. :D
>>
>> Thanks,
>> Gary
>>
>> --
>> Xinran Guo
>> xguo@berkeley.edu
>>
>> On March 10, 2015 at 12:26:24 PM, spdemille-ucb (notifications@github.com)
>> wrote:
>>
>> Hi Gary, I've added create and update subtasks to the calcentral-task-view
>> branch :) still working on delete
>>
>> On Tue, Mar 10, 2015 at 2:12 AM, Sterling P DE MILLE <
>> spdemille@berkeley.edu
>> > wrote:
>>
>> > hi gary, yes, it was without whitespace. it was already that way from
>> the
>> > cal-central website :) i have added a couple more changes if you pull
>> > calcentral-task-view again. i'm still trying to figure out the best way
>> to
>> > do the update/create/delete links for the subtasks. i think i have
>> > something soon :) thx!
>> >
>> > On Mon, Mar 9, 2015 at 10:04 PM, Garyguo2011 <notifications@github.com>
>> > wrote:
>> >
>> >> Hi Sterling,
>> >>
>> >> Did you copy application.css as minimum file? (without any white space)
>> >>
>> >>
>> >> Thanks,
>> >> Gary
>> >>
>> >> --
>> >> Xinran Guo
>> >> xguo@berkeley.edu
>> >>
>> >> On March 9, 2015 at 8:11:56 PM, Xinran Guo (xguo@berkeley.edu) wrote:
>> >>
>> >> Like we discuss last Friday, we probably need to keep edit/update in
>> the
>> >> same page of subtask view.
>> >> and probably a bit move compound around. like Remaining time etc.
>> >>
>> >> Thanks
>> >> Gary
>> >>
>> >> --
>> >> Xinran Guo
>> >> xguo@berkeley.edu
>> >>
>> >> On March 9, 2015 at 7:47:21 PM, spdemille-ucb (
>> notifications@github.com)
>> >> wrote:
>> >>
>> >> what changes need to be done? the files look complete
>> >>
>> >> On Mon, Mar 9, 2015 at 7:14 PM, Garyguo2011 <notifications@github.com>
>> >> wrote:
>> >>
>> >> > Looks nice. Just need some changes for create/update/edit page. :D
>> >> >
>> >> > Thanks for you effort.
>> >> >
>> >> > Gary
>> >> >
>> >> > --
>> >> > Xinran Guo
>> >> > xguo@berkeley.edu
>> >> >
>> >> > On March 9, 2015 at 7:06:14 PM, spdemille-ucb (
>> notifications@github.com
>> >> )
>> >> > wrote:
>> >> >
>> >> > hey gary, i managed to install skype on my laptop which has a camera
>> and
>> >> > microphone :) also, if you pull the calcentral-task-view you can see
>> the
>> >> > CalCentral CSS applied to the task we already have :) i'm working on
>> the
>> >> > task view and trying to figure out how to do graphs. i will push more
>> >> soon
>> >> > :)
>> >> >
>> >> > -- Sterling
>> >> >
>> >> >
>> >> > On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <
>> notifications@github.com>
>> >> > wrote:
>> >> >
>> >> > > Merged #12
>> >> > > <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12>.
>> >> > >
>> >> > > —
>> >> > > Reply to this email directly or view it on GitHub
>> >> > > <
>> >> >
>> >>
>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786
>> >> > >
>> >> >
>> >> > > .
>> >> > >
>> >> > —
>> >> > Reply to this email directly or view it on GitHub.
>> >> >
>> >> > —
>> >> > Reply to this email directly or view it on GitHub
>> >> > <
>> >>
>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77983958
>> >> >
>> >>
>> >> > .
>> >> >
>> >> —
>> >> Reply to this email directly or view it on GitHub.
>> >>
>> >> —
>> >> Reply to this email directly or view it on GitHub
>> >> <
>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77996500
>> >
>>
>> >> .
>> >>
>> >
>> >
>> —
>> Reply to this email directly or view it on GitHub.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-78131241>
>> .
>>
>
>

Garyguo2011 commented:
Hi Sterling,

Wow. I saw that. that is super cool. I just noticed add a new field called END TIME. We didn’t add to DB yet right? Probably we can discuss about it before you add to DB.





-- 
Xinran Guo
xguo@berkeley.edu

On March 10, 2015 at 1:01:48 PM, spdemille-ucb (notifications@github.com) wrote:

i was trying to see if i could make the sub task and task view inside the
center column on the dashboard page of calcentral. i guess i can try
scanning the css document for the column width :)

On Tue, Mar 10, 2015 at 1:00 PM, Sterling P DE MILLE <spdemille@berkeley.edu
> wrote:

> ok, yeah, i texted Roy about it and told him to copy what he needs :) i
> just did a page view source and copy/paste lol :P
>
> On Tue, Mar 10, 2015 at 12:37 PM, Garyguo2011 <notifications@github.com>
> wrote:
>
>> Nice, Thanks for the hard work.
>>
>> Looks fantastic, Specially for the header bar. Super cool. Since you are
>> helping (Daniel, and Roy)’s feature. Probably Your guys need to communicate
>> with each other to avoid duplicate view work.
>>
>> I figure out how to write about Rspec yesterday. will fill test cases
>> correspond to every feature we come up with. i will check your branch. :D
>>
>> Thanks,
>> Gary
>>
>> --
>> Xinran Guo
>> xguo@berkeley.edu
>>
>> On March 10, 2015 at 12:26:24 PM, spdemille-ucb (notifications@github.com)
>> wrote:
>>
>> Hi Gary, I've added create and update subtasks to the calcentral-task-view
>> branch :) still working on delete
>>
>> On Tue, Mar 10, 2015 at 2:12 AM, Sterling P DE MILLE <
>> spdemille@berkeley.edu
>> > wrote:
>>
>> > hi gary, yes, it was without whitespace. it was already that way from
>> the
>> > cal-central website :) i have added a couple more changes if you pull
>> > calcentral-task-view again. i'm still trying to figure out the best way
>> to
>> > do the update/create/delete links for the subtasks. i think i have
>> > something soon :) thx!
>> >
>> > On Mon, Mar 9, 2015 at 10:04 PM, Garyguo2011 <notifications@github.com>
>> > wrote:
>> >
>> >> Hi Sterling,
>> >>
>> >> Did you copy application.css as minimum file? (without any white space)
>> >>
>> >>
>> >> Thanks,
>> >> Gary
>> >>
>> >> --
>> >> Xinran Guo
>> >> xguo@berkeley.edu
>> >>
>> >> On March 9, 2015 at 8:11:56 PM, Xinran Guo (xguo@berkeley.edu) wrote:
>> >>
>> >> Like we discuss last Friday, we probably need to keep edit/update in
>> the
>> >> same page of subtask view.
>> >> and probably a bit move compound around. like Remaining time etc.
>> >>
>> >> Thanks
>> >> Gary
>> >>
>> >> --
>> >> Xinran Guo
>> >> xguo@berkeley.edu
>> >>
>> >> On March 9, 2015 at 7:47:21 PM, spdemille-ucb (
>> notifications@github.com)
>> >> wrote:
>> >>
>> >> what changes need to be done? the files look complete
>> >>
>> >> On Mon, Mar 9, 2015 at 7:14 PM, Garyguo2011 <notifications@github.com>
>> >> wrote:
>> >>
>> >> > Looks nice. Just need some changes for create/update/edit page. :D
>> >> >
>> >> > Thanks for you effort.
>> >> >
>> >> > Gary
>> >> >
>> >> > --
>> >> > Xinran Guo
>> >> > xguo@berkeley.edu
>> >> >
>> >> > On March 9, 2015 at 7:06:14 PM, spdemille-ucb (
>> notifications@github.com
>> >> )
>> >> > wrote:
>> >> >
>> >> > hey gary, i managed to install skype on my laptop which has a camera
>> and
>> >> > microphone :) also, if you pull the calcentral-task-view you can see
>> the
>> >> > CalCentral CSS applied to the task we already have :) i'm working on
>> the
>> >> > task view and trying to figure out how to do graphs. i will push more
>> >> soon
>> >> > :)
>> >> >
>> >> > -- Sterling
>> >> >
>> >> >
>> >> > On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <
>> notifications@github.com>
>> >> > wrote:
>> >> >
>> >> > > Merged #12
>> >> > > <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12>.
>> >> > >
>> >> > > —
>> >> > > Reply to this email directly or view it on GitHub
>> >> > > <
>> >> >
>> >>
>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786
>> >> > >
>> >> >
>> >> > > .
>> >> > >
>> >> > —
>> >> > Reply to this email directly or view it on GitHub.
>> >> >
>> >> > —
>> >> > Reply to this email directly or view it on GitHub
>> >> > <
>> >>
>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77983958
>> >> >
>> >>
>> >> > .
>> >> >
>> >> —
>> >> Reply to this email directly or view it on GitHub.
>> >>
>> >> —
>> >> Reply to this email directly or view it on GitHub
>> >> <
>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77996500
>> >
>>
>> >> .
>> >>
>> >
>> >
>> —
>> Reply to this email directly or view it on GitHub.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-78131241>
>> .
>>
>
>
—
Reply to this email directly or view it on GitHub.

spdemille-ucb commented:
yeah, and i noticed a few things about making a task, like the status
should not be filled in be the user except for pre-determined values like
maybe a dropdown box :) we should all meet and discuss before class maybe

On Tue, Mar 10, 2015 at 1:27 PM, Garyguo2011 <notifications@github.com>
wrote:

> Hi Sterling,
>
> Wow. I saw that. that is super cool. I just noticed add a new field called
> END TIME. We didn’t add to DB yet right? Probably we can discuss about it
> before you add to DB.
>
>
>
>
>
> --
> Xinran Guo
> xguo@berkeley.edu
>
>
> On March 10, 2015 at 1:01:48 PM, spdemille-ucb (notifications@github.com)
> wrote:
>
> i was trying to see if i could make the sub task and task view inside the
> center column on the dashboard page of calcentral. i guess i can try
> scanning the css document for the column width :)
>
> On Tue, Mar 10, 2015 at 1:00 PM, Sterling P DE MILLE <
> spdemille@berkeley.edu
> > wrote:
>
> > ok, yeah, i texted Roy about it and told him to copy what he needs :) i
> > just did a page view source and copy/paste lol :P
> >
> > On Tue, Mar 10, 2015 at 12:37 PM, Garyguo2011 <notifications@github.com>
> > wrote:
> >
> >> Nice, Thanks for the hard work.
> >>
> >> Looks fantastic, Specially for the header bar. Super cool. Since you are
> >> helping (Daniel, and Roy)’s feature. Probably Your guys need to
> communicate
> >> with each other to avoid duplicate view work.
> >>
> >> I figure out how to write about Rspec yesterday. will fill test cases
> >> correspond to every feature we come up with. i will check your branch.
> :D
> >>
> >> Thanks,
> >> Gary
> >>
> >> --
> >> Xinran Guo
> >> xguo@berkeley.edu
> >>
> >> On March 10, 2015 at 12:26:24 PM, spdemille-ucb (
> notifications@github.com)
> >> wrote:
> >>
> >> Hi Gary, I've added create and update subtasks to the
> calcentral-task-view
> >> branch :) still working on delete
> >>
> >> On Tue, Mar 10, 2015 at 2:12 AM, Sterling P DE MILLE <
> >> spdemille@berkeley.edu
> >> > wrote:
> >>
> >> > hi gary, yes, it was without whitespace. it was already that way from
> >> the
> >> > cal-central website :) i have added a couple more changes if you pull
> >> > calcentral-task-view again. i'm still trying to figure out the best
> way
> >> to
> >> > do the update/create/delete links for the subtasks. i think i have
> >> > something soon :) thx!
> >> >
> >> > On Mon, Mar 9, 2015 at 10:04 PM, Garyguo2011 <
> notifications@github.com>
> >> > wrote:
> >> >
> >> >> Hi Sterling,
> >> >>
> >> >> Did you copy application.css as minimum file? (without any white
> space)
> >> >>
> >> >>
> >> >> Thanks,
> >> >> Gary
> >> >>
> >> >> --
> >> >> Xinran Guo
> >> >> xguo@berkeley.edu
> >> >>
> >> >> On March 9, 2015 at 8:11:56 PM, Xinran Guo (xguo@berkeley.edu)
> wrote:
> >> >>
> >> >> Like we discuss last Friday, we probably need to keep edit/update in
> >> the
> >> >> same page of subtask view.
> >> >> and probably a bit move compound around. like Remaining time etc.
> >> >>
> >> >> Thanks
> >> >> Gary
> >> >>
> >> >> --
> >> >> Xinran Guo
> >> >> xguo@berkeley.edu
> >> >>
> >> >> On March 9, 2015 at 7:47:21 PM, spdemille-ucb (
> >> notifications@github.com)
> >> >> wrote:
> >> >>
> >> >> what changes need to be done? the files look complete
> >> >>
> >> >> On Mon, Mar 9, 2015 at 7:14 PM, Garyguo2011 <
> notifications@github.com>
> >> >> wrote:
> >> >>
> >> >> > Looks nice. Just need some changes for create/update/edit page. :D
> >> >> >
> >> >> > Thanks for you effort.
> >> >> >
> >> >> > Gary
> >> >> >
> >> >> > --
> >> >> > Xinran Guo
> >> >> > xguo@berkeley.edu
> >> >> >
> >> >> > On March 9, 2015 at 7:06:14 PM, spdemille-ucb (
> >> notifications@github.com
> >> >> )
> >> >> > wrote:
> >> >> >
> >> >> > hey gary, i managed to install skype on my laptop which has a
> camera
> >> and
> >> >> > microphone :) also, if you pull the calcentral-task-view you can
> see
> >> the
> >> >> > CalCentral CSS applied to the task we already have :) i'm working
> on
> >> the
> >> >> > task view and trying to figure out how to do graphs. i will push
> more
> >> >> soon
> >> >> > :)
> >> >> >
> >> >> > -- Sterling
> >> >> >
> >> >> >
> >> >> > On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <
> >> notifications@github.com>
> >> >> > wrote:
> >> >> >
> >> >> > > Merged #12
> >> >> > > <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12
> >.
> >> >> > >
> >> >> > > —
> >> >> > > Reply to this email directly or view it on GitHub
> >> >> > > <
> >> >> >
> >> >>
> >>
> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786
> >> >> > >
> >> >> >
> >> >> > > .
> >> >> > >
> >> >> > —
> >> >> > Reply to this email directly or view it on GitHub.
> >> >> >
> >> >> > —
> >> >> > Reply to this email directly or view it on GitHub
> >> >> > <
> >> >>
> >>
> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77983958
> >> >> >
> >> >>
> >> >> > .
> >> >> >
> >> >> —
> >> >> Reply to this email directly or view it on GitHub.
> >> >>
> >> >> —
> >> >> Reply to this email directly or view it on GitHub
> >> >> <
> >>
> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77996500
> >> >
> >>
> >> >> .
> >> >>
> >> >
> >> >
> >> —
> >> Reply to this email directly or view it on GitHub.
> >>
> >> —
> >> Reply to this email directly or view it on GitHub
> >> <
> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-78131241
> >
>
> >> .
> >>
> >
> >
> —
> Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-78140844>
> .
>

spdemille-ucb commented:
oh and the end time was the timestamp we talked about for graphing purposes
:)

On Tue, Mar 10, 2015 at 2:37 PM, Sterling P DE MILLE <spdemille@berkeley.edu
> wrote:

> yeah, and i noticed a few things about making a task, like the status
> should not be filled in be the user except for pre-determined values like
> maybe a dropdown box :) we should all meet and discuss before class maybe
>
> On Tue, Mar 10, 2015 at 1:27 PM, Garyguo2011 <notifications@github.com>
> wrote:
>
>> Hi Sterling,
>>
>> Wow. I saw that. that is super cool. I just noticed add a new field
>> called END TIME. We didn’t add to DB yet right? Probably we can discuss
>> about it before you add to DB.
>>
>>
>>
>>
>>
>> --
>> Xinran Guo
>> xguo@berkeley.edu
>>
>>
>> On March 10, 2015 at 1:01:48 PM, spdemille-ucb (notifications@github.com)
>> wrote:
>>
>> i was trying to see if i could make the sub task and task view inside the
>> center column on the dashboard page of calcentral. i guess i can try
>> scanning the css document for the column width :)
>>
>> On Tue, Mar 10, 2015 at 1:00 PM, Sterling P DE MILLE <
>> spdemille@berkeley.edu
>> > wrote:
>>
>> > ok, yeah, i texted Roy about it and told him to copy what he needs :) i
>> > just did a page view source and copy/paste lol :P
>> >
>> > On Tue, Mar 10, 2015 at 12:37 PM, Garyguo2011 <notifications@github.com
>> >
>> > wrote:
>> >
>> >> Nice, Thanks for the hard work.
>> >>
>> >> Looks fantastic, Specially for the header bar. Super cool. Since you
>> are
>> >> helping (Daniel, and Roy)’s feature. Probably Your guys need to
>> communicate
>> >> with each other to avoid duplicate view work.
>> >>
>> >> I figure out how to write about Rspec yesterday. will fill test cases
>> >> correspond to every feature we come up with. i will check your branch.
>> :D
>> >>
>> >> Thanks,
>> >> Gary
>> >>
>> >> --
>> >> Xinran Guo
>> >> xguo@berkeley.edu
>> >>
>> >> On March 10, 2015 at 12:26:24 PM, spdemille-ucb (
>> notifications@github.com)
>> >> wrote:
>> >>
>> >> Hi Gary, I've added create and update subtasks to the
>> calcentral-task-view
>> >> branch :) still working on delete
>> >>
>> >> On Tue, Mar 10, 2015 at 2:12 AM, Sterling P DE MILLE <
>> >> spdemille@berkeley.edu
>> >> > wrote:
>> >>
>> >> > hi gary, yes, it was without whitespace. it was already that way from
>> >> the
>> >> > cal-central website :) i have added a couple more changes if you pull
>> >> > calcentral-task-view again. i'm still trying to figure out the best
>> way
>> >> to
>> >> > do the update/create/delete links for the subtasks. i think i have
>> >> > something soon :) thx!
>> >> >
>> >> > On Mon, Mar 9, 2015 at 10:04 PM, Garyguo2011 <
>> notifications@github.com>
>> >> > wrote:
>> >> >
>> >> >> Hi Sterling,
>> >> >>
>> >> >> Did you copy application.css as minimum file? (without any white
>> space)
>> >> >>
>> >> >>
>> >> >> Thanks,
>> >> >> Gary
>> >> >>
>> >> >> --
>> >> >> Xinran Guo
>> >> >> xguo@berkeley.edu
>> >> >>
>> >> >> On March 9, 2015 at 8:11:56 PM, Xinran Guo (xguo@berkeley.edu)
>> wrote:
>> >> >>
>> >> >> Like we discuss last Friday, we probably need to keep edit/update in
>> >> the
>> >> >> same page of subtask view.
>> >> >> and probably a bit move compound around. like Remaining time etc.
>> >> >>
>> >> >> Thanks
>> >> >> Gary
>> >> >>
>> >> >> --
>> >> >> Xinran Guo
>> >> >> xguo@berkeley.edu
>> >> >>
>> >> >> On March 9, 2015 at 7:47:21 PM, spdemille-ucb (
>> >> notifications@github.com)
>> >> >> wrote:
>> >> >>
>> >> >> what changes need to be done? the files look complete
>> >> >>
>> >> >> On Mon, Mar 9, 2015 at 7:14 PM, Garyguo2011 <
>> notifications@github.com>
>> >> >> wrote:
>> >> >>
>> >> >> > Looks nice. Just need some changes for create/update/edit page. :D
>> >> >> >
>> >> >> > Thanks for you effort.
>> >> >> >
>> >> >> > Gary
>> >> >> >
>> >> >> > --
>> >> >> > Xinran Guo
>> >> >> > xguo@berkeley.edu
>> >> >> >
>> >> >> > On March 9, 2015 at 7:06:14 PM, spdemille-ucb (
>> >> notifications@github.com
>> >> >> )
>> >> >> > wrote:
>> >> >> >
>> >> >> > hey gary, i managed to install skype on my laptop which has a
>> camera
>> >> and
>> >> >> > microphone :) also, if you pull the calcentral-task-view you can
>> see
>> >> the
>> >> >> > CalCentral CSS applied to the task we already have :) i'm working
>> on
>> >> the
>> >> >> > task view and trying to figure out how to do graphs. i will push
>> more
>> >> >> soon
>> >> >> > :)
>> >> >> >
>> >> >> > -- Sterling
>> >> >> >
>> >> >> >
>> >> >> > On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <
>> >> notifications@github.com>
>> >> >> > wrote:
>> >> >> >
>> >> >> > > Merged #12
>> >> >> > > <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12
>> >.
>> >> >> > >
>> >> >> > > —
>> >> >> > > Reply to this email directly or view it on GitHub
>> >> >> > > <
>> >> >> >
>> >> >>
>> >>
>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786
>> >> >> > >
>> >> >> >
>> >> >> > > .
>> >> >> > >
>> >> >> > —
>> >> >> > Reply to this email directly or view it on GitHub.
>> >> >> >
>> >> >> > —
>> >> >> > Reply to this email directly or view it on GitHub
>> >> >> > <
>> >> >>
>> >>
>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77983958
>> >> >> >
>> >> >>
>> >> >> > .
>> >> >> >
>> >> >> —
>> >> >> Reply to this email directly or view it on GitHub.
>> >> >>
>> >> >> —
>> >> >> Reply to this email directly or view it on GitHub
>> >> >> <
>> >>
>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77996500
>> >> >
>> >>
>> >> >> .
>> >> >>
>> >> >
>> >> >
>> >> —
>> >> Reply to this email directly or view it on GitHub.
>> >>
>> >> —
>> >> Reply to this email directly or view it on GitHub
>> >> <
>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-78131241
>> >
>>
>> >> .
>> >>
>> >
>> >
>> —
>> Reply to this email directly or view it on GitHub.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-78140844>
>> .
>>
>
>

spdemille-ucb commented:
the "end time" or timestamp is NOT user edit-able :)

On Tue, Mar 10, 2015 at 2:38 PM, Sterling P DE MILLE <spdemille@berkeley.edu
> wrote:

> oh and the end time was the timestamp we talked about for graphing
> purposes :)
>
> On Tue, Mar 10, 2015 at 2:37 PM, Sterling P DE MILLE <
> spdemille@berkeley.edu> wrote:
>
>> yeah, and i noticed a few things about making a task, like the status
>> should not be filled in be the user except for pre-determined values like
>> maybe a dropdown box :) we should all meet and discuss before class maybe
>>
>> On Tue, Mar 10, 2015 at 1:27 PM, Garyguo2011 <notifications@github.com>
>> wrote:
>>
>>> Hi Sterling,
>>>
>>> Wow. I saw that. that is super cool. I just noticed add a new field
>>> called END TIME. We didn’t add to DB yet right? Probably we can discuss
>>> about it before you add to DB.
>>>
>>>
>>>
>>>
>>>
>>> --
>>> Xinran Guo
>>> xguo@berkeley.edu
>>>
>>>
>>> On March 10, 2015 at 1:01:48 PM, spdemille-ucb (notifications@github.com)
>>> wrote:
>>>
>>> i was trying to see if i could make the sub task and task view inside the
>>> center column on the dashboard page of calcentral. i guess i can try
>>> scanning the css document for the column width :)
>>>
>>> On Tue, Mar 10, 2015 at 1:00 PM, Sterling P DE MILLE <
>>> spdemille@berkeley.edu
>>> > wrote:
>>>
>>> > ok, yeah, i texted Roy about it and told him to copy what he needs :) i
>>> > just did a page view source and copy/paste lol :P
>>> >
>>> > On Tue, Mar 10, 2015 at 12:37 PM, Garyguo2011 <
>>> notifications@github.com>
>>> > wrote:
>>> >
>>> >> Nice, Thanks for the hard work.
>>> >>
>>> >> Looks fantastic, Specially for the header bar. Super cool. Since you
>>> are
>>> >> helping (Daniel, and Roy)’s feature. Probably Your guys need to
>>> communicate
>>> >> with each other to avoid duplicate view work.
>>> >>
>>> >> I figure out how to write about Rspec yesterday. will fill test cases
>>> >> correspond to every feature we come up with. i will check your
>>> branch. :D
>>> >>
>>> >> Thanks,
>>> >> Gary
>>> >>
>>> >> --
>>> >> Xinran Guo
>>> >> xguo@berkeley.edu
>>> >>
>>> >> On March 10, 2015 at 12:26:24 PM, spdemille-ucb (
>>> notifications@github.com)
>>> >> wrote:
>>> >>
>>> >> Hi Gary, I've added create and update subtasks to the
>>> calcentral-task-view
>>> >> branch :) still working on delete
>>> >>
>>> >> On Tue, Mar 10, 2015 at 2:12 AM, Sterling P DE MILLE <
>>> >> spdemille@berkeley.edu
>>> >> > wrote:
>>> >>
>>> >> > hi gary, yes, it was without whitespace. it was already that way
>>> from
>>> >> the
>>> >> > cal-central website :) i have added a couple more changes if you
>>> pull
>>> >> > calcentral-task-view again. i'm still trying to figure out the best
>>> way
>>> >> to
>>> >> > do the update/create/delete links for the subtasks. i think i have
>>> >> > something soon :) thx!
>>> >> >
>>> >> > On Mon, Mar 9, 2015 at 10:04 PM, Garyguo2011 <
>>> notifications@github.com>
>>> >> > wrote:
>>> >> >
>>> >> >> Hi Sterling,
>>> >> >>
>>> >> >> Did you copy application.css as minimum file? (without any white
>>> space)
>>> >> >>
>>> >> >>
>>> >> >> Thanks,
>>> >> >> Gary
>>> >> >>
>>> >> >> --
>>> >> >> Xinran Guo
>>> >> >> xguo@berkeley.edu
>>> >> >>
>>> >> >> On March 9, 2015 at 8:11:56 PM, Xinran Guo (xguo@berkeley.edu)
>>> wrote:
>>> >> >>
>>> >> >> Like we discuss last Friday, we probably need to keep edit/update
>>> in
>>> >> the
>>> >> >> same page of subtask view.
>>> >> >> and probably a bit move compound around. like Remaining time etc.
>>> >> >>
>>> >> >> Thanks
>>> >> >> Gary
>>> >> >>
>>> >> >> --
>>> >> >> Xinran Guo
>>> >> >> xguo@berkeley.edu
>>> >> >>
>>> >> >> On March 9, 2015 at 7:47:21 PM, spdemille-ucb (
>>> >> notifications@github.com)
>>> >> >> wrote:
>>> >> >>
>>> >> >> what changes need to be done? the files look complete
>>> >> >>
>>> >> >> On Mon, Mar 9, 2015 at 7:14 PM, Garyguo2011 <
>>> notifications@github.com>
>>> >> >> wrote:
>>> >> >>
>>> >> >> > Looks nice. Just need some changes for create/update/edit page.
>>> :D
>>> >> >> >
>>> >> >> > Thanks for you effort.
>>> >> >> >
>>> >> >> > Gary
>>> >> >> >
>>> >> >> > --
>>> >> >> > Xinran Guo
>>> >> >> > xguo@berkeley.edu
>>> >> >> >
>>> >> >> > On March 9, 2015 at 7:06:14 PM, spdemille-ucb (
>>> >> notifications@github.com
>>> >> >> )
>>> >> >> > wrote:
>>> >> >> >
>>> >> >> > hey gary, i managed to install skype on my laptop which has a
>>> camera
>>> >> and
>>> >> >> > microphone :) also, if you pull the calcentral-task-view you can
>>> see
>>> >> the
>>> >> >> > CalCentral CSS applied to the task we already have :) i'm
>>> working on
>>> >> the
>>> >> >> > task view and trying to figure out how to do graphs. i will push
>>> more
>>> >> >> soon
>>> >> >> > :)
>>> >> >> >
>>> >> >> > -- Sterling
>>> >> >> >
>>> >> >> >
>>> >> >> > On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <
>>> >> notifications@github.com>
>>> >> >> > wrote:
>>> >> >> >
>>> >> >> > > Merged #12
>>> >> >> > > <
>>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12>.
>>> >> >> > >
>>> >> >> > > —
>>> >> >> > > Reply to this email directly or view it on GitHub
>>> >> >> > > <
>>> >> >> >
>>> >> >>
>>> >>
>>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786
>>> >> >> > >
>>> >> >> >
>>> >> >> > > .
>>> >> >> > >
>>> >> >> > —
>>> >> >> > Reply to this email directly or view it on GitHub.
>>> >> >> >
>>> >> >> > —
>>> >> >> > Reply to this email directly or view it on GitHub
>>> >> >> > <
>>> >> >>
>>> >>
>>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77983958
>>> >> >> >
>>> >> >>
>>> >> >> > .
>>> >> >> >
>>> >> >> —
>>> >> >> Reply to this email directly or view it on GitHub.
>>> >> >>
>>> >> >> —
>>> >> >> Reply to this email directly or view it on GitHub
>>> >> >> <
>>> >>
>>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77996500
>>> >> >
>>> >>
>>> >> >> .
>>> >> >>
>>> >> >
>>> >> >
>>> >> —
>>> >> Reply to this email directly or view it on GitHub.
>>> >>
>>> >> —
>>> >> Reply to this email directly or view it on GitHub
>>> >> <
>>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-78131241
>>> >
>>>
>>> >> .
>>> >>
>>> >
>>> >
>>> —
>>> Reply to this email directly or view it on GitHub.
>>>
>>> —
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-78140844>
>>> .
>>>
>>
>>
>

spdemille-ucb commented:
we can hide that field for now too :)

On Tue, Mar 10, 2015 at 2:39 PM, Sterling P DE MILLE <spdemille@berkeley.edu
> wrote:

> the "end time" or timestamp is NOT user edit-able :)
>
> On Tue, Mar 10, 2015 at 2:38 PM, Sterling P DE MILLE <
> spdemille@berkeley.edu> wrote:
>
>> oh and the end time was the timestamp we talked about for graphing
>> purposes :)
>>
>> On Tue, Mar 10, 2015 at 2:37 PM, Sterling P DE MILLE <
>> spdemille@berkeley.edu> wrote:
>>
>>> yeah, and i noticed a few things about making a task, like the status
>>> should not be filled in be the user except for pre-determined values like
>>> maybe a dropdown box :) we should all meet and discuss before class maybe
>>>
>>> On Tue, Mar 10, 2015 at 1:27 PM, Garyguo2011 <notifications@github.com>
>>> wrote:
>>>
>>>> Hi Sterling,
>>>>
>>>> Wow. I saw that. that is super cool. I just noticed add a new field
>>>> called END TIME. We didn’t add to DB yet right? Probably we can discuss
>>>> about it before you add to DB.
>>>>
>>>>
>>>>
>>>>
>>>>
>>>> --
>>>> Xinran Guo
>>>> xguo@berkeley.edu
>>>>
>>>>
>>>> On March 10, 2015 at 1:01:48 PM, spdemille-ucb (
>>>> notifications@github.com) wrote:
>>>>
>>>> i was trying to see if i could make the sub task and task view inside
>>>> the
>>>> center column on the dashboard page of calcentral. i guess i can try
>>>> scanning the css document for the column width :)
>>>>
>>>> On Tue, Mar 10, 2015 at 1:00 PM, Sterling P DE MILLE <
>>>> spdemille@berkeley.edu
>>>> > wrote:
>>>>
>>>> > ok, yeah, i texted Roy about it and told him to copy what he needs :)
>>>> i
>>>> > just did a page view source and copy/paste lol :P
>>>> >
>>>> > On Tue, Mar 10, 2015 at 12:37 PM, Garyguo2011 <
>>>> notifications@github.com>
>>>> > wrote:
>>>> >
>>>> >> Nice, Thanks for the hard work.
>>>> >>
>>>> >> Looks fantastic, Specially for the header bar. Super cool. Since you
>>>> are
>>>> >> helping (Daniel, and Roy)’s feature. Probably Your guys need to
>>>> communicate
>>>> >> with each other to avoid duplicate view work.
>>>> >>
>>>> >> I figure out how to write about Rspec yesterday. will fill test cases
>>>> >> correspond to every feature we come up with. i will check your
>>>> branch. :D
>>>> >>
>>>> >> Thanks,
>>>> >> Gary
>>>> >>
>>>> >> --
>>>> >> Xinran Guo
>>>> >> xguo@berkeley.edu
>>>> >>
>>>> >> On March 10, 2015 at 12:26:24 PM, spdemille-ucb (
>>>> notifications@github.com)
>>>> >> wrote:
>>>> >>
>>>> >> Hi Gary, I've added create and update subtasks to the
>>>> calcentral-task-view
>>>> >> branch :) still working on delete
>>>> >>
>>>> >> On Tue, Mar 10, 2015 at 2:12 AM, Sterling P DE MILLE <
>>>> >> spdemille@berkeley.edu
>>>> >> > wrote:
>>>> >>
>>>> >> > hi gary, yes, it was without whitespace. it was already that way
>>>> from
>>>> >> the
>>>> >> > cal-central website :) i have added a couple more changes if you
>>>> pull
>>>> >> > calcentral-task-view again. i'm still trying to figure out the
>>>> best way
>>>> >> to
>>>> >> > do the update/create/delete links for the subtasks. i think i have
>>>> >> > something soon :) thx!
>>>> >> >
>>>> >> > On Mon, Mar 9, 2015 at 10:04 PM, Garyguo2011 <
>>>> notifications@github.com>
>>>> >> > wrote:
>>>> >> >
>>>> >> >> Hi Sterling,
>>>> >> >>
>>>> >> >> Did you copy application.css as minimum file? (without any white
>>>> space)
>>>> >> >>
>>>> >> >>
>>>> >> >> Thanks,
>>>> >> >> Gary
>>>> >> >>
>>>> >> >> --
>>>> >> >> Xinran Guo
>>>> >> >> xguo@berkeley.edu
>>>> >> >>
>>>> >> >> On March 9, 2015 at 8:11:56 PM, Xinran Guo (xguo@berkeley.edu)
>>>> wrote:
>>>> >> >>
>>>> >> >> Like we discuss last Friday, we probably need to keep edit/update
>>>> in
>>>> >> the
>>>> >> >> same page of subtask view.
>>>> >> >> and probably a bit move compound around. like Remaining time etc.
>>>> >> >>
>>>> >> >> Thanks
>>>> >> >> Gary
>>>> >> >>
>>>> >> >> --
>>>> >> >> Xinran Guo
>>>> >> >> xguo@berkeley.edu
>>>> >> >>
>>>> >> >> On March 9, 2015 at 7:47:21 PM, spdemille-ucb (
>>>> >> notifications@github.com)
>>>> >> >> wrote:
>>>> >> >>
>>>> >> >> what changes need to be done? the files look complete
>>>> >> >>
>>>> >> >> On Mon, Mar 9, 2015 at 7:14 PM, Garyguo2011 <
>>>> notifications@github.com>
>>>> >> >> wrote:
>>>> >> >>
>>>> >> >> > Looks nice. Just need some changes for create/update/edit page.
>>>> :D
>>>> >> >> >
>>>> >> >> > Thanks for you effort.
>>>> >> >> >
>>>> >> >> > Gary
>>>> >> >> >
>>>> >> >> > --
>>>> >> >> > Xinran Guo
>>>> >> >> > xguo@berkeley.edu
>>>> >> >> >
>>>> >> >> > On March 9, 2015 at 7:06:14 PM, spdemille-ucb (
>>>> >> notifications@github.com
>>>> >> >> )
>>>> >> >> > wrote:
>>>> >> >> >
>>>> >> >> > hey gary, i managed to install skype on my laptop which has a
>>>> camera
>>>> >> and
>>>> >> >> > microphone :) also, if you pull the calcentral-task-view you
>>>> can see
>>>> >> the
>>>> >> >> > CalCentral CSS applied to the task we already have :) i'm
>>>> working on
>>>> >> the
>>>> >> >> > task view and trying to figure out how to do graphs. i will
>>>> push more
>>>> >> >> soon
>>>> >> >> > :)
>>>> >> >> >
>>>> >> >> > -- Sterling
>>>> >> >> >
>>>> >> >> >
>>>> >> >> > On Mon, Mar 9, 2015 at 4:24 PM, Garyguo2011 <
>>>> >> notifications@github.com>
>>>> >> >> > wrote:
>>>> >> >> >
>>>> >> >> > > Merged #12
>>>> >> >> > > <
>>>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12>.
>>>> >> >> > >
>>>> >> >> > > —
>>>> >> >> > > Reply to this email directly or view it on GitHub
>>>> >> >> > > <
>>>> >> >> >
>>>> >> >>
>>>> >>
>>>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#event-248959786
>>>> >> >> > >
>>>> >> >> >
>>>> >> >> > > .
>>>> >> >> > >
>>>> >> >> > —
>>>> >> >> > Reply to this email directly or view it on GitHub.
>>>> >> >> >
>>>> >> >> > —
>>>> >> >> > Reply to this email directly or view it on GitHub
>>>> >> >> > <
>>>> >> >>
>>>> >>
>>>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77983958
>>>> >> >> >
>>>> >> >>
>>>> >> >> > .
>>>> >> >> >
>>>> >> >> —
>>>> >> >> Reply to this email directly or view it on GitHub.
>>>> >> >>
>>>> >> >> —
>>>> >> >> Reply to this email directly or view it on GitHub
>>>> >> >> <
>>>> >>
>>>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-77996500
>>>> >> >
>>>> >>
>>>> >> >> .
>>>> >> >>
>>>> >> >
>>>> >> >
>>>> >> —
>>>> >> Reply to this email directly or view it on GitHub.
>>>> >>
>>>> >> —
>>>> >> Reply to this email directly or view it on GitHub
>>>> >> <
>>>> https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-78131241
>>>> >
>>>>
>>>> >> .
>>>> >>
>>>> >
>>>> >
>>>> —
>>>> Reply to this email directly or view it on GitHub.
>>>>
>>>> —
>>>> Reply to this email directly or view it on GitHub
>>>> <https://github.com/Garyguo2011/CalCentral-Task-Planner/pull/12#issuecomment-78140844>
>>>> .
>>>>
>>>
>>>
>>
>
